hongit-api/src/main/kotlin/com/devfloor/hongit/api/article/presentation/ArticleController.kt

```diff
@@ -31,16 +33,43 @@ class ArticleController(
     @GetMapping(value = ["/{articleId}"])
     @ResponseStatus(value = HttpStatus.OK)
     fun showByArticleId(@PathVariable articleId: Long): ArticleResponse = articleService.showByArticleId(articleId)
+        .also { log.info("[ArticleController.showByArticleId] 게시글 조회 완료 - , response: $it") }
```

Comment:
```suggestion
        .also { log.info("[ArticleController.showByArticleId] 게시글 조회 완료 - response: $it") }
```

아래 다른 로그도 그런데 이 ,는 의도한 콤마?인건가??